test/integration/features/provision_earlier_version.feature

```diff
@@ -12,6 +13,17 @@ Feature: Provision an older major release
      openshift v1.5.1
      """
 
+  @cdk-only
+  Scenario: Starting Minishift with ocp v3.5
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --ocp-tag v3.5" succeeds
+     Then Minishift should have state "Running"
+     When executing "minishift openshift version" succeeds
+     Then stdout should contain
```

Comment:
Please use step for matching regular expression as the OCP 3.5 can get update and then this would need an update also. Could be maybe something like this: `openshift v3\.5\.[5-9]\.[0-9]{1,2}\.[0-9]{1,3}`

Reply:
Done